c2cgeoform/scaffolds/c2cgeoform/acceptance_tests/excavation_test.py_tmpl

```diff
@@ -89,11 +103,17 @@ class TestExcavationViews():
         resp = test_app.get("/excavations/{}".format(exc.hash), status=200)
 
         form = resp.form
-        # assert form["request_date"] == exc.request_date
+        assert exc.reference_number == form['reference_number'].value
+        assert exc.hash == form['hash'].value
+        assert str(exc.id) == form.fields.get('id')[0].value
+        assert str(exc.request_date) == resp.html.find('input', attrs={'name': 'date'}).attrs['value']
```

Comment:
why not `form['date'].value` ? why `date` and not `request_date` ?

Reply:
There is one and only one date in the form, and targetting value with css selector

'''
</div>
    <div
     class="form-group  item-request_date item-request_date"
     title=""
     id="item-deformField4">
  <label for="deformField4"
         class="control-label "
         id="req-deformField4"
         >
    Request Date
  </label> 
    <input type="hidden" name="__start__" value="request_date:mapping"/>
  <input type="date"
         name="date"
         value="2017-07-31"
         id="deformField4" class="  form-control hasDatepicker"/>
  <input type="hidden" name="__end__" value="request_date:mapping"/>
  <script type="text/javascript">
   deform.addCallback(
    'deformField4',
     function deform_cb(oid) {
       if (!Modernizr.inputtypes['date'] ||"date" != "date" || window.forceDateTimePolyfill){
         $('#' + oid).pickadate({"selectYears": true, "formatSubmit": "yyyy-mm-dd", "selectMonths": true, "format": "yyyy-mm-dd"});
       }
     }
   );
  </script>   
</div>
'''